pyhealth/datasets/eicu.py

```diff
@@ -238,7 +239,15 @@ def icd9cm_or_icd10cm(code):
                 return "ICD10CM"
             else:
                 return "Unknown"
-
+            
+        # prefer icd9 code when possible, fallback to the icd9 code
```

Comment:
this comment is incorrect; we only return the icd9 equivalent. For synthetic data generation, we want to keep the dataset as homogenous as possible. Maybe in the future we will support mixed code vocabulary.